### ISSUE TITLE
[Feature Adjustment] Hotbar Frames on per bar basis

### ISSUE THREAD

**Issue by someONEgood** [UnknownX7/QoLBar#34]
**Hello and thank you again for this wonderful plugin.**
TLDR: Would it be possible to allow enabling **[Use Hotbar Frames on Icons]** on a per bar (or even category) basis?
Alternatively, disable it for Spacer buttons.

I was wondering if this would be possible to implement?
I love the high gloss design. However after checking out a recent shared-plugin setting, I noticed that you couldn't set them on a per bar basis, resulting in this:
![jobselect](https://user-images.githubusercontent.com/46905309/137597166-142bdb0f-f08a-4d09-902d-45ae56ab77f0.JPG)

[Plugin by Hold Shift For the Love of God: [https://discord.com/channels/581875019861328007/795684290750971945/898735903651020810](url)]

Thank you for your consideration.
